api/swagger/forms.yaml

```diff
@@ -4,12 +4,13 @@ info:
   description:  |
     AEM Forms as a Cloud Service - Headless Forms HTTP APIs.
     # Introduction
-    Headless APIs let you fetch the Headless Adaptive Form definitions, and submit form data to AEM.
+    Headless APIs let you get the Headless Adaptive Form definition, prefill the form, validate and submit it.
   version: '1.0.0'
 servers:
-  - url: '/adobe/forms/af/v1'
+  - url: '/adobe/forms/af'
 externalDocs:
   description: API documentation
+  url: "https://wiki.corp.adobe.com/display/lc/Form+Headless+API+Specification"
```

Comment:
minor: let's remove internal url's from the public documentation, and point to a more appropriate accessible endpoint.